colorbleed/plugins/maya/publish/collect_renderlayers.py

```diff
@@ -39,6 +38,11 @@ def process(self, context):
                         cmds.getAttr("{}.renderable".format(i)) and not
                         cmds.referenceQuery(i, isNodeReferenced=True)]
 
+        # Sort by displayOrder
+        renderlayers = sorted(renderlayers,
+                              key=lambda x: -cmds.getAttr("%s.displayOrder" % x))
+
+        renderlayers = reversed(renderlayers)
```

Comment:
Actually this is double? Did you add this reversed? Because the `negative` sign in front of displayOrder should already reverse it. :)